### PR TITLE
Adopt popup window

### PR DESCRIPTION
**Pull request by get-me-power**
I adopted popup window in this plugin when user choice episode. Please review this code.

### CODE REVIEW

**Comment by mattn** [comment]
Thanks. s/epsode/episode/g

**Comment by get-me-power** [comment]
Sorry. I fixed.

File: plugin/starwars.vim [hunk 1]
```diff
@@ -1,6 +1,12 @@
+scriptencoding utf-8
+
 let s:dir = expand('<sfile>:h:h') . '/resources/'
 function! s:play(...) abort
-  let l:ep = get(a:000, 0, 1)
+  if has("patch-8.1.1453")
+    let l:ep = a:1
+  else
+    let l:ep = get(a:000, 0, 1)
+  endif
```
Comment (mattn) [inline plugin/starwars.vim:7]: What do you mean?

https://github.com/vim/vim/commit/3397f74ac2ac27f1eef48e950c3c8eeb0338fe55

Comment (get-me-power) [reply plugin/starwars.vim:7]: Sorry, I was misunderstood so I fixed.

I noticed that `let l:ep = a:1` is my mistake so I deleted this sentence.

**Comment by kazukazuinaina** [comment]
I Fixed.

**Comment by get-me-power** [comment]
By the way, I think `if has("patch-8.1.1453")` mean  vim can judge use or not use popup method, but I noticed that same mean `if exists('*popup_create')`.  Should I fix this syntax in last line?

## Before

```
if has("patch-8.1.1453")
  command -nargs=0 StarWars call starwars#selectepisode()
else
  command -nargs=? StarWars call s:play(<f-args>)
endif
```

## After

```
if exists('*popup_create')
  command -nargs=0 StarWars call starwars#selectepisode()
else
  command -nargs=? StarWars call s:play(<f-args>)
endif
```

Which is better ?

**Comment by mattn** [comment]
`starwars#selectepisode()` should be called in `s:play` when no arguments are given for s:play.

**Comment by get-me-power** [comment]
I fixed. Please check this code.

File: plugin/starwars.vim [hunk 2]
```diff
@@ -2,39 +2,43 @@ scriptencoding utf-8
...
-    call setline(1, l:frame['lines'])
-    redraw!
-    let l:duration = max([l:frame['duration'] * 1000 / l:speed, 1])
-    exe printf('sleep %dms', l:duration)
-    let l:key = getchar(0)
-    if l:key == 106
-      if l:speed > 1
-        let l:speed += 1
+  if len(a:000) == 0 && exists('*popup_create')
+    call starwars#selectepisode()
```
Comment (mattn) [inline plugin/starwars.vim]: early return & ommit nesting, please

**Comment by get-me-power** [comment]
Sorry... I fixed.

**Comment by mattn** [comment]
Merged. Thank you.

https://github.com/mattn/vim-starwars/commits/master

**Pull request closed without merging**